SharedMemory/RpcBuffer.cs

```diff
@@ -973,8 +973,16 @@ void ReadThreadV1()
                             // For Handling Request we create an new Task because this can take sometime
                             Task.Run(async () =>
                             {
-                                await ProcessCallHandler(request).ConfigureAwait(false);
-                            });    
+                                try
+                                {
+                                    await ProcessCallHandler(request).ConfigureAwait(false);
+                                }
+                                catch
```

Comment:
If this truly is about the ObjectDisposed scenario then this should only be catching `System.ObjectDisposedException`. Do you agree?

Reply:
Yes thats right, maybe also `InvalidOperationException` for shutting down, right?